client/src/InfoBase/index_data.js

```diff
@@ -95,7 +95,7 @@ const index_lang_lookups = {
   },
   bonus_footer_list: {
     en: `
```

Comment:
Heads up on here, French version also need to be changed just below !

Reply:
Good catch! I've implemented the fixes and I'm waiting to get the translation checked